pkg/cloud/ibmcloud/options/options.go

```diff
@@ -22,9 +22,11 @@ import (
 )
 
 var (
-	TokenTTL time.Duration
+	TokenTTL               time.Duration
+	WaitTransactionTimeout int
```

Comment:
Is there any reason to use `int` instead of `time.Duration`? I saw you convert `int` to `Duration` later for checking.

Reply:
Use duration instead of int